webapp/channels/src/sass/utils/_mixins.scss

```diff
@@ -74,7 +74,6 @@
 
 @mixin clearfix {
     overflow: hidden;
-    *zoom: 1;
```

Comment:
star property hack used for ie7 and below

Reply:
Are we supporting IE7 with this? Can we not just remove it?

Reply:
we were supporting ie7 with that, thats why i removed it since we are no longer supporting any version of ie

Reply:
Ahh the remnants always linger :P